modules/@amperka/octoliner.js

```diff
@@ -9,31 +9,38 @@ var Octoliner = function(opts) {
     opts.i2c = I2C1;
   }
   this.expander = require('@amperka/gpio-expander').connect(opts);
-  this.expander.pwmFreq(60000);
+  this.expander.pwmFreq(8000);
   this._sensePin = 0;
   this._ledBrightnessPin = 9;
+  this.expander.analogWrite(this._ledBrightnessPin, 1);
   this._sensorPinMap = [4, 5, 6, 8, 7, 3, 2, 1];
   this._lineState = [];
   this.setSensitivity(0.91);
-  this.setBrightness(1);
+  this.mapPatternToLine = this.defaultMapPatternToLine;
+  this.mapAnalogToPattern = this.defaultMapAnalogToPattern;
```

Comment:
Why so? Just define `mapPatternToLine`, `mapAnalogToPattern`. A user may override them using regular prototypal JS-inheritance.

Reply:
It seems so clearer to me.

Reply:
This is not a native way for JS. It can confuse people who use JS in the JS-way. Suggest making the methods regular.